cabal.project

```diff
@@ -0,0 +1,8 @@
+packages: .
+
+source-repository-package
```

Comment:
Temporary source-repository-package.
I will remove it after HsYAML-0.2 is released.